maestro-cli/src/main/java/maestro/cli/cloud/CloudInteractor.kt

```diff
@@ -275,6 +279,16 @@ class CloudInteractor(
                 throw CliError("Failed to fetch the status of an upload $uploadId. Status code = ${e.statusCode}")
             }
 
+            for (uploadFlowResult in upload.flows) {
+                if(printedFlows.contains(uploadFlowResult)) { continue }
+                if(!terminalStatuses.contains(uploadFlowResult.status)) { continue }
+
+                printedFlows.add(uploadFlowResult);
+                TestSuiteStatusView.showFlowCompletion(
+                  uploadFlowResult.toViewModel()
+                )
```

Comment:
Can we add a test for cloud interactor that checks its output especially for local.